envoy-control-core/src/main/kotlin/pl/allegro/tech/servicemesh/envoycontrol/groups/MetadataNodeGroup.kt

```diff
@@ -176,6 +180,16 @@ class MetadataNodeGroup(
     }
 }
 
+interface CustomMetadataMapper {
```

Comment:
i think that we dont need any mapper class, we can assume that customData is send in `customData` property and is Map<String, Any>

Reply:
It's not that simple. Because when we received customData they are wrapped up in to protobuf objects. So we have 3 options:
 - store raw protobuf data
 - write generic mapper from raw protobuf
 - use this mapper class

I don't like storing raw protobuf, so maybe we should create generic mapper